celery_once/helpers.py

```diff
@@ -34,6 +34,22 @@ def now_unix():
     return int(time())
 
 
+def force_string(kwargs):
+    """
+    Force key in dict or list to a string.
+    Fixes: https://github.com/TrackMaven/celery-once/issues/11
+    """
+    if isinstance(kwargs, dict):
+        return {
+            force_string(key): force_string(value) for key, value in six.iteritems(kwargs)}
+    elif isinstance(kwargs, list):
+        return [force_string(element) for element in kwargs]
+    elif isinstance(kwargs, six.string_types) or isinstance(kwargs, six.text_type) or isinstance(kwargs, six.binary_type):
+        return str(kwargs)
```

Comment:
I would instead do the following
```python
elif six.PY2 and isinstance(kwargs, unicode):
    return kwargs.encode('utf-8')
```
As the "u" prefix is only used in python2, for unicode, and ``str(kwargs)`` would fail in python2 if ``kwargs`` contains non ascii characters, as the default encoding is ascii.

```python
In [8]: str(u"héhé")
---------------------------------------------------------------------------
UnicodeEncodeError                        Traceback (most recent call last)
<ipython-input-8-60c755478a13> in <module>()
----> 1 str(u"héhé")

UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 1: ordinal not in range(128)
```

What do you think?